FCCee/ALLEGRO/compact/ALLEGRO_o1_v03/materials.xml

```diff
@@ -475,4 +475,35 @@
     <fraction n="0.084" ref="Nb"/>
   </material>
 
+
+  <material name="Water">
+      <D value="1" unit="g/cm3" />
+      <composite n="2" ref="H" />
+      <composite n="1" ref="O" />
+  </material>
+
+  <material name="DefaultMaterial">
```

Comment:
Default material for what?

Reply:
I realized that if there isn't a DefaultMaterial the CAD import may fail for some reason, as it will look for this material even if the assigned one is different. In this case I have DefaultMaterial = steel. In the log you can also see this:
```
ASSIMPReader     INFO  +++ "Componente1"     Material: DefaultMaterial   Viualization: NONE
ASSIMPReader           +++ Read 1 meshes from /afs/cern.ch/user/a/aciarma/public/git_k4geo/aciarma_CADpipe2/k4geo/FCCee/MDI/compact/MDI_o1_v01/stl_files/Pipe_240430/AlBeMet162_30042024.stl
TestShape        INFO  Created successfull shape of type: CAD_MultiVolume [CAD_MultiVolume]
TestShape        INFO  Correct shape type: CAD_MultiVolume CAD_MultiVolume <-> CAD_MultiVolume Instance test: OK
```
but inspecting the volumes with `dd4hep2root` and https://root.cern/js/latest/ you can see it is actually AlBeMet (the same for all the other CAD volumes).